include/hipSYCL/runtime/code_object_invoker.hpp

```diff
@@ -61,6 +61,20 @@ class sscp_code_object_invoker {
                                std::size_t *arg_sizes, std::size_t num_args,
                                const std::string &kernel_name,
                                const glue::kernel_configuration& config) = 0;
+
+  virtual rt::range<3> select_group_size(const rt::range<3> &num_groups,
```

Comment:
It seems the way that this function is used, actually the first parameter is the global size, not the number of groups. This confused me for a while when I tried to understand the min/max logic that the OpenMP backend implements this function with. Perhaps rename the parameter?